src/_variables.scss

```diff
@@ -9,8 +9,11 @@ $default-palette: $palette;
 $green-palette: palette(
     $primary: #09f,
     $secondary: #72da67,
-    $grays: #fff,
-    $surface: #333
+    $surface: #333,
```

Comment:
You're trying to get the gray color shade from this palette but there isn't any gray color passed to the palette.

Reply:
That's fine, the gray color is automatically generated when not provided based on the surface color.